### PR TITLE
[WORKFLOWS-553] add orca-recipes for Sarek pipeline

### PR DESCRIPTION
**Pull request by danlu1**
Problem:

There is no Sarek Version3 recipe.

Solution:

Add orca-recipes for Sarek pipeline including nf-synapse/NF_SYNSTAGE, nf-core/sarek, and synindex. The pipeline has run through successfully for one sample already: [nf-synapse/NF_SYNSTAGE](https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/ntap-add5-project/watch/50Qa9WCps4oBDn), [nf-core/sarek](https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/ntap-add5-project/watch/5Dgk6huJZsGJJW), and synindex(https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/ntap-add5-project/watch/5zUZ1TJkwqfUmF).

Note:
Sorry for the confusing branch name. I used the wrong ticket number here. It's actually WORKFLOWS-553 instead of WORKFLOWS-560

### CODE REVIEW

File: local/ntap/workflows-553-sarek.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+"""
+import asyncio
+import boto3
+
+from synapseclient import Synapse
+from dataclasses import dataclass
+from pathlib import Path
+
+from orca.services.nextflowtower import NextflowTowerOps
+from orca.services.nextflowtower.models import LaunchInfo
```
Comment (BWMac) [inline local/ntap/workflows-553-sarek.py]: NIT: Incorrect sorting/unused import (Path). Consider using the (https://pycqa.github.io/isort/) VSCode extension (if you use VSCode).

**Review by BWMac — APPROVED** [review]
LGTM! Did this script work all the way through all 3 Tower workflow executions? Can you provide links to the workflow runs in the PR description?

File: local/ntap/workflows-553-sarek.py (hunk 1 above)
Comment (danlu1) [reply local/ntap/workflows-553-sarek.py]: Thanks for catching it. I was copying this from an existing script and didn't check.

**Comment by danlu1** [comment]
> LGTM! Did this script work all the way through all 3 Tower workflow executions? Can you provide links to the workflow runs in the PR description?

@BWMac Not yet, It runs through the stage step for now. And I'm going to run other steps for one sample and let the NF team check the output. Let me make this a draft PR for now.

**Review by BWMac — APPROVED** [review]
🚀

**Pull request merged**